php/class-plugin.php

```diff
@@ -304,7 +304,7 @@ public function process_post_query_vars( $post_query_vars ) {
 		);
 
 		// White list allowed meta query compare values.
-		$allowed_meta_query_compare_values = array( '=', '!=', '>', '>=', '<', '<=' );
+		$allowed_meta_query_compare_values = array( '=', '!=', '>', '>=', '<', '<=', 'LIKE' );
```

Comment:
@westonruter My filter looks like this
```
add_filter( 'customize_object_selector_post_query_vars', function ( $post_query_vars ) {
	if ( ! empty( $post_query_vars['dropdown_args']['has_some_param'] ) ) {
		$search_term = $post_query_vars['s'];

		$post_query_vars['meta_query'] = array(
			array(
				'key' => 'my_meta_key',
				'value'   => $search_term,
				'compare' => 'LIKE',
			),
			array(
				'key' => 'my_meta_key',
				'value'   => '',
				'compare' => '!=',
			),
		);

		$post_query_vars['s'] = '';
	}

	return $post_query_vars;
} );
```

where I want to search by meta key, and if `LIKE` is not included in `$allowed_meta_query_compare_values` it would throw error of `disallowed_meta_compare_query_var`. I was not able to understand how your snippet would achieve that from a plugin. Can you explain a bit more please?